package.json

```diff
@@ -22,11 +22,11 @@
     "lint": "eslint . --cache --max-warnings 0",
     "lint-fix": "eslint . --fix",
     "typecheck": "tsc && tsc -p integration_tests",
-    "test": "jest -i",
+    "test": "jest -i --coverage",
```

Comment:
Turned coverage report on for `npm test` - though could add it as a separate task if you prefer....?

Reply:
Fine with me here 👍